entrypoint.sh

```diff
@@ -21,16 +21,36 @@ URI=https://api.github.com
 API_HEADER="Accept: application/vnd.github.v3+json"
 AUTH_HEADER="Authorization: token $GITHUB_TOKEN"
 
-pr_resp=$(curl -X GET -s -H "${AUTH_HEADER}" -H "${API_HEADER}" \
-          "${URI}/repos/$GITHUB_REPOSITORY/pulls/$PR_NUMBER")
+MAX_RETRIES=${MAX_RETRIES:-6}
+RETRY_INTERVAL=${RETRY_INTERVAL:-10}
+REBASEABLE=""
+pr_resp=""
+for ((i = 0 ; i < $MAX_RETRIES ; i++)); do
+	pr_resp=$(curl -X GET -s -H "${AUTH_HEADER}" -H "${API_HEADER}" \
+		"${URI}/repos/$GITHUB_REPOSITORY/pulls/$PR_NUMBER")
+	REBASEABLE=$(echo "$pr_resp" | jq -r .rebaseable)
+	if [[ "$REBASEABLE" == "null" ]]; then
+		echo "The PR is not ready to rebase, retry after $RETRY_INTERVAL seconds"
+		sleep $RETRY_INTERVAL
+		continue
+	else
+		break
+	fi
+done
+
+if [[ "$REBASEABLE" != "true" || "$i" == "$MAX_RETRIES" ]] ; then
```

Comment:
Also no need to check for `i` value. Not even sure if it's still available here. 🤔 

```suggestion
if [[ "$REBASEABLE" != "true" ]] ; then
```

Reply:
https://github.com/cirrus-actions/rebase/pull/75/commits/e4987e76cfabed2bbe500c3b855a435b58cf7947